src/bundle/DependencyInjection/Configuration/Parser/AdminUiForms.php

```diff
@@ -84,6 +88,35 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
                             ->end()
                         ->end()
                     ->end()
+                    ->arrayNode('content_type_edit')
+                        ->info('Content Type Edit form configuration')
+                        ->children()
+                            ->scalarNode('default_meta_field_type_group')
+                                ->info('Group name used to add meta field types')
+                                ->beforeNormalization()
+                                    ->ifTrue(
+                                        static function (string $groupName): bool {
+                                            return empty($groupName) || !preg_match(self::GROUP_NAME_PATTERN, $groupName);
+                                        }
+                                    )
+                                    ->thenInvalid('The group name "%s" contains illegal characters. Group names should start with a letter, digit or underscore and only contain letters, digits, numbers, underscores ("_"), hyphens ("-") and colons (":").')
+                                ->end()
+                            ->end()
+                            ->arrayNode('fieldtypes')
```

Comment:
Naming: while it's not been consistent across the codebase and repo names, the proper form would be `field_types` given these are documented as 2 words.

```suggestion
                            ->arrayNode('field_types')
```